python/mxnet/foo/nn/conv_layers.py

```diff
@@ -158,6 +158,22 @@ class Conv1D(_Conv):
         see Initializer.
     bias_initializer: Initializer for the bias vector
         see Initializer.
+
+
+    Input Shape:
```

Comment:
![screen shot 2017-06-26 at 4 57 00 pm](https://user-images.githubusercontent.com/8004078/27565056-8300ffa8-5a90-11e7-80ee-80f2ba80a831.png)
This is how it looks